### DISCUSSION TITLE
Cant get github student pack

### DISCUSSION THREAD

**Discussion by egerdnc** [community/community#52454] [GitHub Education] [answered]
Hi,

Since the last 7.2 Earthquake incident in my home country Turkey, all of the Highschools in my area and also all of the Universities in the whole country change their education system to Online Education and because of that, I am unable to get my School ID Card from my Highschool.

Even tho I can still get a paper online from the Ministry of Education online portal.
That paper shows the active studentship status of the person, you can see it by translating the given picture.

For some reason Github Student Pack system wants me to only upload my School ID and doesn't accept any other kind of proof. (I can still pick other types of proof from the box, but the system doesn't accept any.)

Im willing to share a Ministry of Education approved paper of mine.

I would love to continue using Github Student Pack benefits.

**Comment by samus-aran** [comment, marked as answer]
Hi @egerdnc 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. 

Hope this helps! Let us know how you get along 👍

**Reply by egerdnc** [reply]
Is there any way to do it manually?
Because I'm not sure how I would get the English translation of the official paper.
And I doubt the system would accept that.

**Reply by samus-aran** [reply]
Yes, you can manually translate important pieces of information yourself and show it along side your original. We would suggest you pick the important parts e.g. school's name, date of studying, course information, your details like name, email address etc.

**Comment by quanboy247** [comment]
> Hi,
> 
> Since the last 7.2 Earthquake incident in my home country Turkey, all of the Highschools in my area and also all of the Universities in the whole country change their education system to Online Education and because of that, I am unable to get my School ID Card from my Highschool.
> 
> Even tho I can still get a paper online from the Ministry of Education online portal. That paper shows the active studentship status of the person, you can see it by translating the given picture.
> 
> For some reason Github Student Pack system wants me to only upload my School ID and doesn't accept any other kind of proof. (I can still pick other types of proof from the box, but the system doesn't accept any.)
> 
> Im willing to share a Ministry of Education approved paper of mine.
> 
> I would love to continue using Github Student Pack benefits.

**Comment by Jamolov-Isroilbek** [comment]
Hi, I would like to know how long it takes to get a response to the request for Student Package. I submitted my Student ID but still didn't get any information regarding it.

**Comment by Alexpn69** [comment]
Hello!
I applied for the GitHub Education program, but my application was rejected. I attached a document stating that I am currently studying to become a web3 developer. I really enjoy this field, and the student pack would be very helpful to me. You can see the results of my studies on my GitHub profile.
I would greatly appreciate it if you could reconsider your decision and include me in the program.
Thank you in advance!
